addons/binding/pom.xml

```diff
@@ -74,6 +74,7 @@
     <module>org.openhab.binding.rfxcom.test</module>
     <module>org.openhab.binding.russound</module>
     <module>org.openhab.binding.samsungtv</module>
+    <module>org.openhab.binding.satel</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).

Reply:
Good to know. Why is it not included in the archetype?

Reply:
Done.